testing/boost-build/APKBUILD

```diff
@@ -1,38 +1,35 @@
 # Contributor: Valery Kartel <valery.kartel@gmail.com>
 # Maintainer: Valery Kartel <valery.kartel@gmail.com>
 pkgname=boost-build
-pkgver=1.62.0
-pkgrel=1
+pkgver=1.66.0
+pkgrel=0
 pkgdesc="Boost.Build makes it easy to build C++ projects, everywhere"
 url="http://boost.org/build"
 arch="all"
+options="!check"
 license="custom:boost"
 depends="boost-dev"
 makedepends=""
 subpackages="$pkgname-doc"
 source="$pkgname-$pkgver.tar.gz::https://github.com/boostorg/${pkgname#*-}/archive/${pkgname%%-*}-$pkgver.tar.gz
-	$pkgname-$pkgver-LICENSE.txt::http://boost.org/LICENSE_1_0.txt"
+	$pkgname-LICENSE.txt::http://boost.org/LICENSE_1_0.txt"
```

Comment:
Why did you change this?

Reply:
Because license file not assigned to package version and not changes between versions.

Reply:
That does not matter. What matters is that source names must be unique among all aports in all versions. If Boost change this file or release new version and you keep the current name, it will fail on builders because sources are cached and identified only by name.